src/components/shared/VirtualList.js

```diff
@@ -345,17 +345,24 @@ export class VirtualList<Item> extends React.PureComponent<
   /**
    * Scroll the container horizontally if necessary
    */
-  _scrollContainerHorizontally(container: HTMLDivElement, offsetX: CssPixels) {
+  _scrollContainerHorizontally(
+    container: HTMLDivElement,
+    itemX: CssPixels,
+    offsetX: CssPixels
+  ) {
     const interestingWidth = 400;
-    const itemLeft = offsetX;
-    const itemRight = itemLeft + interestingWidth;
+    const itemLeft = itemX;
+    const itemRight = itemX + interestingWidth;
```

Comment:
I'm curious why we don't add the `offsetX` to `itemRight` here. It looks like we do the same operation twice below (first we remove the offsetX from the other side of the comparison and then add it later).

Reply:
I think `itemRight` itself should stay like this, as this represents properly the right edge for the item, but I can introduce another variable that would be `itemRight - (container.clientWidth - offsetX)`, just not sure what its ideal name is. `container.clientWidth - offsetX` itself could be called `scrollingColumnWidth`, so maybe `itemRight - scrollingColumnWidth` is `minimumLeftScrollingToSeeRightEdge`? I'm not sure :D I'll think about this some more.

Reply:
Well, I think it's not a big deal. So happy to land it like this too.

Reply:
I decided to only add `scrollingColumnWidth` because I thought the reasoning process was easier to follow with keeping the calculations, but I also added some more comments. Hopefully this should make it easier to reason when looking at this in the future :-)